Revert/Sources/AutoLayoutMarginsViewController.swift

```diff
@@ -23,25 +58,106 @@ final class AutoLayoutMarginsViewController: RevertViewController {
   }
 
   // MARK: Private
-
   private static let interSquareSpacing: CGFloat = 20
 
   private var lastUpdateSquaresWidthSize: CGSize?
 
   @IBOutlet private weak var centerView: UIView!
-  @IBOutlet private weak var slider: UISlider!
+  
   @IBOutlet private weak var centerViewWidthConstraint: NSLayoutConstraint!
   @IBOutlet private weak var containerView: UIView!
   @IBOutlet private weak var containerViewBottomConstraint: NSLayoutConstraint!
 
-  @IBAction private func sliderValueChanged(sender: UISlider) {
-    let margin = CGFloat(sender.value)
-    self.centerView.layoutMargins = UIEdgeInsets(top: margin, left: margin, bottom: margin, right: margin)
-  }
-
   private func updateSquareWidths() {
     let minDistance = min(self.containerView.bounds.width, self.containerView.bounds.height)
     let centerWidth = (minDistance - (4 * self.dynamicType.interSquareSpacing)) / 3
     self.centerViewWidthConstraint.constant = centerWidth
   }
 }
+
+#if os(tvOS)
+final class ButtonsAutoLayoutMarginsViewController: AutoLayoutMarginsViewController {
```

Comment:
Rather than subclassing `AutoLayoutMarginsViewController` you could extract the UI element for manipulating the margin values into a `UIView` subclass.  These two `UIView` subclass could implement a protocol so that the `AutoLayoutMarginsViewController` doesn't need to know the concrete subclass.  Then either with target action or a delegate you could communicate the increment/decrement actions.